doc/CONTRIBUTING.md

```diff
@@ -82,7 +81,7 @@ the report as well, and they have been updated accordingly.
 1. Make sure to add proper details to the Issue raised
 1. Upload screenshot(if possible) in dashboard issues
 1. Apply proper labels to the Issue
-1. Make sure to add Milestone and Project to the issue to mention specific release.
+1. XXXX Make sure to add Milestone and Project to the issue to mention specific release where the issue was encountered.
```

Comment:
Typically, the project's release engineer adds the `Milestone` and `Project`, so we should probably drop this step entirely.